sql/hive/pom.xml

```diff
@@ -59,13 +59,6 @@
     </dependency>
     <dependency>
       <groupId>org.apache.spark</groupId>
-      <artifactId>spark-sql_${scala.binary.version}</artifactId>
```

Comment:
Not strictly related, just found this redundancy while editing. The exact same stanza appears below.